### DISCUSSION TITLE
How to apply style to only Icon on IconButton Hover

### DISCUSSION THREAD

**Discussion by nicolls1** [chakra-ui/chakra-ui#2804] [Q&A] [answered]
Hi!

I'm looking to scale an icon on hover but not the whole button. If I put the transform in the style IconButton _hover then the whole button is transformed rather than just the icon I would like. The current state and my second attempt was to put the Icon in a box and then have a _hover on the box but this would not transform on button hover, just when the icon is.

https://codesandbox.io/s/happy-platform-qxrci?file=/src/index.tsx

What would be the correct way to do something like this with chakra-ui?

Thanks!

**Comment by chr-ge** [comment, marked as answer]
You can move the `_hover` inside the Icon:
```jsx
<IconButton
  aria-label="Right"
  fontSize="15px"
  icon={<ArrowForwardIcon _hover={{ transform: "scale(1.5)" }} />}
  borderRadius="full"
  _hover={{ bgColor: "orange.500" }}
/>
```
If you want to make the Icon scale on Button hover you can do this:
```jsx
<IconButton
  aria-label="Right"
  fontSize="15px"
  icon={
   <Center _hover={{ transform: "scale(1.5)" }} h="100%" w="100%">
     <ArrowForwardIcon />
   </Center>
  }
  borderRadius="full"
  _hover={{ bgColor: "orange.500" }}
/>
```

**Reply by nicolls1** [reply]
The Center with `h="100%" w="100%"` works great! Thanks!
